graylog2-server/src/main/java/org/graylog2/indexer/messages/Messages.java

```diff
@@ -164,210 +75,47 @@ public ResultMessage get(String messageId, String index) throws DocumentNotFound
-                            (int) Math.ceil((double) messageList.size() / chunkSize), offset);
-                }
-                LOG.debug("Index: Bulk indexed {} messages{}, failures: {}",
-                        result.getItems().size(), chunkInfo, failedItems.size());
-            }
-            if (!remainingFailures.isEmpty()) {
-                LOG.error("Failed to index [{}] messages. Please check the index error log in your web interface for the reason. Error: {}",
-                        remainingFailures.size(), result.getErrorMessage());
-            }
-            chunkCount++;
-        }
-        return failedItems;
-    }
-
-    private BulkResult bulkIndexChunk(List<Map.Entry<IndexSet, Message>> chunk) {
-        Bulk.Builder bulk = new Bulk.Builder();
-
-        for (Map.Entry<IndexSet, Message> entry : chunk) {
-            final Message message = entry.getValue();
-
-            bulk.addAction(new Index.Builder(message.toElasticSearchObject(invalidTimestampMeter))
-                    .index(entry.getKey().getWriteIndexAlias())
-                    .type(IndexMapping.TYPE_MESSAGE)
-                    .id(message.getId())
-                    .build());
-        }
-
-        return runBulkRequest(bulk.build(), chunk.size());
+        return propagateFailure(indexFailures);
     }
 
-    private Set<BulkResult.BulkResultItem> retryOnlyIndexBlockItemsForever(List<Map.Entry<IndexSet, Message>> chunk, List<BulkResult.BulkResultItem> allFailedItems) {
-        Set<BulkResult.BulkResultItem> indexBlocks = indexBlocksFrom(allFailedItems);
-        final Set<BulkResult.BulkResultItem> otherFailures = new HashSet<>(Sets.difference(new HashSet<>(allFailedItems), indexBlocks));
-        List<Map.Entry<IndexSet, Message>> blockedMessages = messagesForResultItems(chunk, indexBlocks);
-
-        if (!indexBlocks.isEmpty()) {
-            LOG.warn("Retrying {} messages, because their indices are blocked with status [read-only / allow delete]", indexBlocks.size());
-        }
-
-        long attempt = 1;
-
-        while (!indexBlocks.isEmpty()) {
-            waitBeforeRetrying(attempt++);
-
-            final BulkResult bulkResult = bulkIndexChunk(blockedMessages);
-
-            final List<BulkResult.BulkResultItem> failedItems = bulkResult.getFailedItems();
-
-            indexBlocks = indexBlocksFrom(failedItems);
-            blockedMessages = messagesForResultItems(blockedMessages, indexBlocks);
-
-            final Set<BulkResult.BulkResultItem> newOtherFailures = Sets.difference(new HashSet<>(failedItems), indexBlocks);
-            otherFailures.addAll(newOtherFailures);
-
-            if (indexBlocks.isEmpty()) {
-                LOG.info("Retries were successful after {} attempts. Ingestion will continue now.", attempt);
-            }
-        }
+    private void accountTotalMessageSizes(List<IndexingRequest> successfullRequests, boolean isSystemTraffic) {
```

Comment:
Now that we reproduced the old behavior the parameter should just be named `request`.